core/include/drivers/stm32_bsec.h

```diff
@@ -170,11 +170,13 @@ bool stm32_bsec_nsec_can_access_otp(uint32_t otp_id);
  * Find and get OTP location from its name.
  * @name: sub-node name to look up.
  * @otp_id: pointer to output OTP number or NULL.
+ * @otp_bit_offset: pointer to output OTP offset in bits or NULL.
```

Comment:
` * @otp_bit_offset: pointer to output OTP bit offset in the NVMEM cell or NULL.`